client/flutter/lib/generated/intl/messages_all.dart

```diff
@@ -22,14 +22,14 @@ import 'messages_en.dart' as messages_en;
 import 'messages_ru.dart' as messages_ru;
 import 'messages_fr.dart' as messages_fr;
 
-typedef Future<dynamic> LibraryLoader();
+typedef LibraryLoader = Future<dynamic> Function();
```

Comment:
This is written within the **generated** folder

Reply:
@theswerd I know. You do not need to point out every file. This is due to `flutter format`. The command *does not allow* excluding directories.

I will update my original comment to explain my considerations.